### ISSUE TITLE
Support for non-GIT repositories

### ISSUE THREAD

**Issue by NVentimiglia** [d12frosted/UnityLibrarySwitcher#8]
This asset causes exceptions when you using SVN. Would you consider making the GIT requirement optional ?

**Comment by d12frosted** [comment]
Yes, currently only git is supported, that's why it's not working for other VCSs. But it's easy to add support for SVN, Mercurial or whatever else. 

I am quite busy lately, so it will take probably two weeks for me to start working on this request, so if you need it sooner - feel free to contribute, PRs are welcome :)
